NEWS

```diff
@@ -1,4 +1,4 @@
-VennDiagram 1.8.0 2023-08-23 (Stefan Eng, Dan Knight)
+VennDiagram 1.8.0 2023-08-28 (Stefan Eng, Dan Knight)
```

Comment:
Add yourself!

Reply:
The package doesn't have an `Authors@R` field in DESCRIPTION where I could add myself, and apparently `Maintainer` can only be a single person: "The mandatory ‘Maintainer’ field should give a single name followed by a valid (RFC 2822) email address in angle brackets" ([Writing R Extensions](https://cran.r-project.org/doc/manuals/R-exts.html#The-DESCRIPTION-file)). I can add the `Authors@R` field or just stop with the addition to NEWS. I probably don't need to be considered a contributor after just deleting a file, so I'd be fine with just that.